website/source/docs/providers/aws/r/redshift_cluster.html.markdown

```diff
@@ -34,10 +34,11 @@ string.
  If you do not provide a name, Amazon Redshift will create a default database called `dev`.
 * `node_type` - (Required) The node type to be provisioned for the cluster.
 * `cluster_type` - (Optional) The cluster type to use. Either `single-node` or `multi-node`. 
-* `master_password` - (Required) Password for the master DB user. Note that this may
-    show up in logs, and it will be stored in the state file. Password must contain at least 8 chars and 
+* `master_password` - (Required unless a `snapshot_identifier` is provided) Password for the master DB user. 
+    Note that this may show up in logs, and it will be stored in the state file. Password must contain at least 8 chars and 
```

Comment:
`master_password` has `Sensitive: true` set, which should keep it's contents out of the logs. We should optionally pgp encrypt it in the state, like we been doing for some other resources